db-migrations/30_versions_python_version.up.sql

```diff
@@ -0,0 +1,5 @@
+-- Introduce Python version column, with default as version 3.7 which is the
+-- only supported major version as of the introduction of the column.
+ALTER TABLE versions ADD COLUMN python_version varchar(16) NOT NULL DEFAULT '3.7.*';
```

Comment:
I think it can be
```
ALTER TABLE versions ADD COLUMN python_version varchar(16) NOT NULL DEFAULT '3.7.*' WITH VALUES;
```

And no need to remove the default in line 5 ?